Resources/Textures/Objects/Devices/communication.rsi/meta.json

```
@@ -94,6 +94,9 @@
     },
     {
       "name": "old-radio-borg-assault"
+    },
+    {
+      "name": "old-radio-bomb-roach"
```

```suggestion
      "name": "old-radio-bombroach"
```
minor formatting
(change the png too)